core/stores/sqlc/cachedsql.go

```diff
@@ -88,6 +88,10 @@ func (cc CachedConn) GetCacheCtx(ctx context.Context, key string, v any) error {
 	return cc.cache.GetCtx(ctx, key, v)
 }
 
+func(cc CachedConn)GetRawConnNoCache()sqlx.SqlConn{
```

Comment:
When NewConn is used, the conn is passed in from the outside. It is of little significance to provide RawConn here. If you really want to get it, you can save it by the business itself, or explain the value of providing the new method.

Reply:
your explanation is reasonable,but this situation need be  token care .In the most of environment, i can only declare a cacheConn variable in my servercontext obviously,but according to the real environment,i must to use the conn which skip the cache level in some condition query,because the series query will  hit rarely. To make easier operation,i must expose this sqlx.SqlConn to directly use the NonCache query.

Reply:
To keep the implementation simple, I think it's more straightforward that you keep the reference on your own.